data/item.go

```diff
@@ -51,6 +53,10 @@ const (
 	TROLL_TOES          = "Troll toes"
 )
 
+func IsSword(name string) bool {
```

Comment:
Is this function ever going to be reused? Can we just move it to Items IsSword?

Reply:
The purpose of putting this function in this place is because constants for item names are defined just above.